mpact/models.py

```diff
@@ -33,9 +37,6 @@ class Chat(ChatBase):
     Represents a telegram group
     """
     title = models.TextField()
-    created_at = models.DateTimeField()
-    start_date = models.DateField(default=timezone.now)
-    start_time = models.TimeField(default=timezone.now)
```

Comment:
This is really strange to me. Hopefully we'll see during our QA process, but makes me nervous that they might have been used somewhere. 

Am I right that it is used in [services.py](https://github.com/dimagi/mpact/blob/77ee70e9d723ff6c56b6c14f77aa56a533744db4/mpact/services.py#L280)? I just did a quick find-in-files to look for `start_date`

Reply:
Oh, actually I don't think these are meant to be duplicates. I think these are meant to be the time of day to send automated messages to the GroupChat (`start_time`) and the starting date to begin stepping through the scheduled messages (`start_date`).

Is that possible?

Reply:
Oops, thanks for the flag. You're right and I'm not sure why that wasn't picked up when I searched (i'll blame getting used to my new dev environment, though might have just been careless).

I added them back and renamed them. Didn't bother fixing the migration history since there's no prod yet. Thinking we can likely aim to just squash all migrations together prior to delivering the "final" v1 codebase. But let me know if you think that'd be problematic